vignettes/download_jira_issues.Rmd

```diff
@@ -0,0 +1,344 @@
+---
+title: "Download JIRA Issues and Comments"
+output: 
+  html_document:
+    toc: true
+    number_sections: true
+vignette: >
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteIndexEntry{Download JIRA Issues and Commentss}
+  %\VignetteEncoding{UTF-8}
+---
+
+
+```{r}
+rm(list = ls())
+seed <- 1
+set.seed(seed)
+```
+
+```{r warning=FALSE,message=FALSE}
+require(kaiaulu)
+require(data.table)
+require(knitr, quietly = T)
+require(dplyr, quietly = T)
+require(jsonlite)
+```
+
+# Introduction
+
+This example is adapted from the JirAgileR package [README.md](https://github.com/matbmeijer/JirAgileR). 
+
+As usual, the first step is to load the project configuration file. 
+
+# Project Configuration File
+
+In this notebook, we will obtain data from the issue tracker JIRA. We will use [Apache's Geronimo open source project](https://geronimo.apache.org). Refer to the `conf/` folder on Kaiaulu's git repository for Geronimo and other project configuration files. It is in this project configuration file we specify where Kaiaulu can find the jira sources from Geronimo. We will use the issue_tracker -> jira fields only. In regards to the "issues" and "issue_comments" fields, these should be set to paths where you want to store the jira data. Then, you can access this jira data later using these same paths.
+
+# your_api_token is your atlassian token that is used as your password. Your username is your atlassian username (usually your email). These can be created by navigating to the indicated directories and creating a plain text file (vim atlassian_username or vim atlassian_token) and typing in your username and token. Details on how to obtain an atlassian token can be found here: https://support.atlassian.com/atlassian-account/docs/manage-api-tokens-for-your-atlassian-account/
```

Comment:
This makes it sound the api token is required. You should clarify it is an optional field, especially when ASF is blocking users to create account on their issue tracker.